File: CCC_2022/CCC_2022_Problem_J3_Harp_Tuning.py
s = input()
tmp = ""
for i in range(len(s)):
    if len(tmp) != 0 and not s[i].isnumeric() and s[i-1].isnumeric():
        print(tmp)
        tmp = "";
    if s[i].isupper():
        tmp += s[i]
    elif s[i] == '-':
        tmp += " loosen "
    elif s[i] == '+':
        tmp += " tighten "
    elif s[i].isnumeric():
        tmp += s[i]
print(tmp)

File: CCC_2022/CCC_2022_Problem_J3_Harp_Tuning.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    string s, tmp = "";
    cin >> s;
    for(int i=0; i<s.size(); i++){
        if(tmp.size()!=0 && !isdigit(s[i]) && isdigit(s[i-1])){
            cout << tmp << endl;
            tmp = "";
        }
        if(isupper(s[i]) || isdigit(s[i])){
            tmp += s[i];
        }
        else if(s[i] == '-'){
            tmp += " loosen ";
        }
        else if(s[i] == '+'){
            tmp += " tighten ";
        }
    }
    cout << tmp << endl;
    return 0;
}
